### ISSUE TITLE
Requisition for Preprocessing Files

### ISSUE THREAD

**Issue by WangJiuniu** [enry12/adversarial_training_methods#1]
Hello Enrico，
Thank you for your code, and it really helps me a lot. And I want to learn more about your code. So, could you please tell me how to get the Preprocessing Files (i.e. nltk_xtrain.npy, nltk_ytrain.npy and nltk_ultrain.npy).
Thanks a lot for your help!
wang
Email: wangjiuniu99@163.com

**Comment by aurelpjetri** [comment]
Hello @WangJiuniu ,
we added the code you requested with the new commit. In those new files there is the code for:
- dataset preprocessing (dataset_script.pynb and test_dataset_script.ipynb)
- dataset preparation for GloVe training (glove_training_preparation.ipynb)
- (launch GloVe training) 
- creation of the embedding matrix used by the network (pretrained_glove_embedding_script.ipynb)

We are not 100% sure about the order of execution of these scripts because it has passed some time since the last time we worked on this project. However we hope it still helps you learn more about our code.
Feel free to ask for additional information.
Best regards,
Aurel and Enrico.
